hail/python/hail/docs/getting_started.rst

```diff
@@ -106,11 +119,13 @@ can run Hail backed by the cluster can be started with the following command::
 
     ipython
 
-When using ``ipython``, you can import hail and start interacting directly
+When using ``ipython``, you can import hail and start interacting directly:
+
+.. code-block:: python
 
     >>> import hail as hl
-    >>> mt = hl.balding_nichols_model(3, 100, 100)
-    >>> mt.aggregate_entries(hl.agg.mean(mt.GT.n_alt_alleles()))
+    >>> mt = hl.balding_nichols_model(3, 50, 100)
```

Comment:
Looks good! Just wondering about the changes from 100->50 and removal of aggregation. Goal was to reduce time to first result?

Reply:
It's not about speed. Making the number of samples and variants distinct better illustrates what the parameters do. I've now added the parameter names as well to be even clearer.

Reply:
And the simplification to count is because the point here is NOT to confuse the hell out of everyone before they've had a chance to learn a bit about Hail usage.